src/languages/en.ts

```diff
@@ -2667,6 +2667,12 @@ export default {
             smartLimitWarning: (limit: string) => `If you change this card’s limit to ${limit}, new transactions will be declined until you approve more expenses on the card.`,
             monthlyLimitWarning: (limit: string) => `If you change this card’s limit to ${limit}, new transactions will be declined until next month.`,
             fixedLimitWarning: (limit: string) => `If you change this card’s limit to ${limit}, new transactions will be declined.`,
+            changeCardLimitType: 'Change card limit type',
+            changeLimitType: 'Change limit type',
```

Comment:
It was [confirmed](https://github.com/Expensify/App/issues/44328#issuecomment-2237241706) to use `Change limit type`, since it's what we change with this button